src/codegeneration/InstantiateModuleTransformer.js

```diff
@@ -368,10 +370,12 @@ export class InstantiateModuleTransformer extends ModuleTransformer {
       // exported dependencies
       if (exportStarBinding) {
         setterStatements = setterStatements.concat(parseStatements `
+          var exportObj = {};
```

Comment:
Yeah, this one needs to be `Object.create(null)`. If not a, an export named `__proto__` will screw everything up.